lib/styles/jquery.js

```diff
@@ -23,7 +23,9 @@ module.exports = function (lines, options, errors) {
   }
 
   var components = options.components
-  if (components.length && components.indexOf(component) === -1) {
+  if (components.length && !components.some(
+    function (x) { return RegExp(x).test(component) }
```

Comment:
Parsing everything as RegExp makes this more lenient. `components: [ "Build" ]` will now accept anything before and after "Build" as well. We could add `^` and `$` to this, that would still allow specifying any valid regex inbetween.